agent/agent.go

```diff
@@ -2,77 +2,77 @@ package agent
 
 import (
 	"context"
-	"crypto/sha256"
 	_ "embed"
-	"encoding/hex"
-	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"log"
-	"math/rand"
-	"net"
-	"net/http"
-	"os"
-	"strconv"
 	"time"
 
-	"github.com/labstack/echo/v4"
-	"github.com/sirupsen/logrus"
-
 	"github.com/libp2p/go-libp2p"
 	"github.com/libp2p/go-libp2p-core/host"
 	"github.com/libp2p/go-libp2p-core/peer"
 	pubsub "github.com/libp2p/go-libp2p-pubsub"
 	configLibp2p "github.com/libp2p/go-libp2p/config"
-	"github.com/multiformats/go-multiaddr"
+	"github.com/maticnetwork/libp2p-gossip-bench/network"
+	ma "github.com/multiformats/go-multiaddr"
 )
 
-//go:embed latencies.json
-var latencyDataRaw string
+const (
+	// overlay parameters
+	gossipSubD   = 8  // topic stable mesh target count
```

Comment:
Could you put these values in the `AgentConfig` and put default values over there. Later on we will change this config during the `main.go` test runner.

Reply:
This also helps to understand from a quick glance all the values that are related to the GossipSub protocol itself.

Reply:
Everything is moved to configuration. Fixed